bpf/lib/conntrack.h

```diff
@@ -167,6 +167,20 @@ static __always_inline bool ct_entry_alive(const struct ct_entry *entry)
 	return !entry->rx_closing || !entry->tx_closing;
 }
 
+static __always_inline bool ct_entry_closing(const struct ct_entry *entry)
+{
+	return entry->tx_closing || entry->rx_closing;
+}
+
+static __always_inline bool
+ct_entry_closing_wait_before_rebalance(const struct ct_entry *entry)
+{
+	__u32 now = bpf_mono_now();
+	__u32 wait_time = bpf_sec_to_mono(CT_SERVICE_CLOSE_WAIT_BEFORE_REBALANCE);
+	return READ_ONCE(entry->last_tx_report) + wait_time >= now ||
+		READ_ONCE(entry->last_rx_report) + wait_time >= now;
```

Comment:
`READ_ONCE(entry->last_tx_report) + wait_time >= now` is there twice?

I think just looking at `last_tx_report` is fine and must in this case given rx side may not be bumped, but probably needs a comment.